views/includes/save-and-continue-button.html

```diff
@@ -1,6 +1,6 @@
 {% set textContinueButton = "Continue" %}
 
-{% if OE_CONFIGS.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 == "true" %}
+{% if OE_CONFIGS.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 == "true" or OE_CONFIGS.FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME == "true" %}
```

Comment:
If registration sets FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 to false, then it will still show "Save and continue" in the registration journey if the Update feature flag is true (and vice versa)

Reply:
We are not planning to go live without save & resume for Update. Can we just revert this change to be only under the OE_CONFIGS.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 flag? For testing just update we can then toggle the flags to align the button with feature but would need some rework if we wanted to go live without save and resume (alongside resume using save & resume). If that's not acceptable we could maybe use {{ page.url }} in the template and check startsWith("update-") or startWith("register-") and factor that into the feature logic. Otherwise would need to pass in from controllers.

Reply:
I think the button logic should be the same as the logic in the 'save.and.continue' util, so probably have to pass something into the controller. Just thinking about when we go live with both update and register save and continue, if there is an issue in live with save and resume, we plan on switching the feature flag off to revert back to the old way of saving while we fix it. If update is also live then setting FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 to false would show the button as just 'continue' in update but it would still save.  This might be a minor issue but my gut feeling is the button logic should be tied to the save.and.continue util logic.

Reply:
@bantao-ch has agreed to make the button fix as a separate bug so happy to mark this as resolved